opencog/nlp/relex2logic/rule-utils.scm

```diff
@@ -6,54 +6,54 @@
 ;--------------------------------------------------------------------
 ;
 ; Short-hand for declaring a variable.
-(define (var-decl var type)
+(define-public (var-decl var type)
```

Comment:
In the long-run, these should be moved out of relex2logic/rule-utils, to some generic nlp/rule-utils file, somewhere.